python/desc/imsim/imSim_class_factory.py

```diff
@@ -52,8 +57,16 @@ def imSim__init__(self, phosim_objects, obs_metadata, catalog_db=None):
     self.db_obj = type('DummyDB', (), dict(epoch=2000))
 
     # Add noise and sky background
-    self.noise_and_background = ExampleCCDNoise(addNoise=True,
-                                                addBackground=True)
+    # The simple code using the default lsst-GalSim interface would be:
+    #
+    #    self.noise_and_background = ExampleCCDNoise(addNoise=True,
+    #                                                addBackground=True)
+    #
+    # But, we need a more realistic sky model and we need to pass more than
+    # this basic info to use Peter Y's ESO sky model.
+    # We must pass obs_metadata, chip information etc...
+    self.noise_and_background = ESOSkyModel(obs_metadata, addNoise=True,
```

Comment:
So, in practice it would show up here right:

phoSimStarCatalog = desc.imsim.ImSimStars(starDataBase, obs_md)

and would look something like 

phoSimStarCatalog = desc.imsim.ImSimStars(starDataBase, obs_md, PSF=special_PSF)

This sounds OK except it seems to me that that PSF is not really a feature of the source and you will have separate entries like above for Galaxies etc.  Perhaps it should be part of the common obs_metadata (which is already repeated)?